ProductList.jsx

```diff
@@ -2,12 +2,23 @@ import React from 'react';
 import Product from './Product.jsx';
 
 export default class ProductList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {products: props.products};
+  }
+  componentDidMount() {
+    $.getJSON('http://localhost:3000/products', function(data) {
```

Comment:
1) We should probably assign the host url to a const outside of the component. Easier to change, easier to read.
2) The callback should use es6 function syntax so we can avoid the `.bind(...)` altogether.